subjects/java/checkpoints/singleton-blueprint/README.md

```diff
@@ -0,0 +1,44 @@
+## Singleton Blueprint
+
+### Instructions
+
+You are given an incomplete `Singleton` class. Complete the class to demonstrate your understanding of how the `Singleton` design pattern works. The `Singleton` pattern ensures that a class has only one instance and provides a global point of access to that instance.
+
+### Expected Class
+
+```java
+public class Singleton {
+    public Singleton instance;
+
+    private Singleton() {
+    }
+
+    public Singleton get???() {
```

Comment:
In the tests, the method called ist `getInstance`: why here is `get???` ?

Reply:
I hide the name they have to know it by them selves.